diff-hl-test.el

```diff
@@ -0,0 +1,121 @@
+;;; diff-hl.el --- Highlight uncommitted changes using VC -*- lexical-binding: t -*-
```

Comment:
Could you put the test file (and the test data) into a new directory? Call it `test`, I guess.